public/css/user/home.css

```diff
@@ -3,6 +3,266 @@
   padding: 0;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+section {
+  height: 100vh;
+}
+
+.main--header {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.main--header-title {
+  margin-top: 150px;
+  width: 100%;
+  display: flex;
+}
+
+.main--header-titleWhite {
+  font-weight: 300;
+  font-family: Roboto;
+  display: flex;
+  background: rgb(0, 131, 38);
+  color: white;
+  width: 500px;
+  height: 200px;
+  justify-content: center;
+  align-items: center;
+  font-size: 72px;
+}
+
+.main--header-titleGreen {
+  font-weight: 300;
+  font-family: Roboto;
+  display: flex;
+  background: white;
+  color: #008326;
+  width: 550px;
+  height: 200px;
+  justify-content: center;
+  align-items: center;
+  font-size: 72px;
+}
+
+.main--header-star {
+  position: absolute;
+  left: 80%;
+  top: 10%;
+  width: 27%;
+  height: 52%;
+}
+
+.main--header-starGreen {
+  position: absolute;
+  clip-path: polygon( 50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
+  width: 500px;
+  height: 500px;
+  background: rgba(0, 131, 38, 0.9);
+  z-index: 1;
+}
+
+.main--header-starGray {
+  position: absolute;
+  clip-path: polygon( 50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
+  transform: matrix(0.97, -0.23, 0.23, 0.97, 0, 0);
+  width: 500px;
+  height: 500px;
+  background: #c4c4c4;
+}
+
+.main--header-description {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 200px;
+}
+
+.main--header-description-p {
+  width: 800px;
+  height: 120px;
+  font-family: lato;
+  font-size: 26px;
+  margin-left: 80px;
+  font-weight: 300;
+}
+
+.main--header-description-request {
+  align-self: flex-end;
+  margin-right: 100px;
+  margin-top: 150px;
+}
+
+.main--header-forYou, .main--header-helpOthers {
+  background: #008326;
+  font-size: 30px;
+  width: 200px;
+  height: 60px;
+  border: none;
+  border-radius: 5px;
+}
+
+.main--header-forYouA, .main--header-helpOthersA {
+  color: white;
+  text-decoration: none;
+}
+
+.main--header-ourVision {
+  background-image: linear-gradient( to left bottom, rgba(1, 1, 1, 0.5), rgba(1, 1, 1, 0.5)), url("https://files.gitter.im/ahmedisam99/pYeg/donation.jpg");
```

Comment:
@aminalakhsham Not super urgent but you could move that jpeg file into this repo

Reply:
do u see that good ...
it size 5 mb so big i think the url more good

Reply:
You can move it into the repo and still use a url, like `background-image: (.. gradients), url('/public/images/donations.jpg);`
It's not too important / urgent but then you're protected against glitter.im having an outage and messing up your website's appearance :)